pkg/river/parser/testdata/invalid_object_key.river

```diff
@@ -0,0 +1,8 @@
+obj {
+  map = {
+    "string_field"    = "foo", 
+    identifier_string = "bar", 
+    1337 /* ERROR "expected field name \(string or identifier\), got NUMBER" */ = "baz", 
```

Comment:
Can you add another valid field after this to make sure it doesn’t report any more errors?